docs/en/connector-v2/source/Iceberg.md

```diff
@@ -71,25 +71,26 @@ libfb303-xxx.jar
+| stream_scan_strategy     | enum     | no       | FROM_LATEST_SNAPSHOT | Starting strategy for stream mode execution, Default to use `FROM_LATEST_SNAPSHOT` if don’t specify any value,The optional values are:<br/>TABLE_SCAN_THEN_INCREMENTAL: Do a regular table scan then switch to the incremental mode.<br/>FROM_LATEST_SNAPSHOT: Start incremental mode from the latest snapshot inclusive.<br/>FROM_EARLIEST_SNAPSHOT: Start incremental mode from the earliest snapshot inclusive.<br/>FROM_SNAPSHOT_ID: Start incremental mode from a snapshot with a specific id inclusive.<br/>FROM_SNAPSHOT_TIMESTAMP: Start incremental mode from a snapshot with a specific timestamp inclusive. |
+| increment.scan-interval  | long     | no       | 2000                 | The interval of increment scan(mills)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| common-options           |          | no       | -                    | Source plugin common parameters, please refer to [Source Common Options](../source-common-options.md) for details.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| compaction_action        | boolean  | no       | false                | Whether to enable the Compaction operation on the Iceberg table. Compaction is an optimization operation that merges small files into larger ones to improve read performance. When set to `true`, the Iceberg connector will perform Compaction during data writes. The default value is `false`, meaning Compaction is not enabled.                                                                                                                                                                                                                                                                                  |
```

Comment:
Hi @sunxiaojian , why we can set `compaction_action` during read from iceberg? Would it do file compaction even read from iceberg?